docs/03-extensions/external-login-handlers/providers/facebook.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Facebook"
+description: "Allow customers to use your website with their Facebook account."
+---
+
+<p>{frontMatter.description}</p>
+
+## Facebook setup
+
+The `FacebookProvider` needs the `clientId` and the `clientSecret` to work. You
+can get these from your app on the
+[Facebook developer console](https://developers.facebook.com/apps/).
+
+### Configuring Redirect URIs
+
+You need to configure the allowed redirect URIs on your
+[Facebook developer console](https://developers.facebook.com/apps/). Note that
+the `Valid OAuth Redirect URIs` should contain all your stores URLs followed by
```

Comment:
**question:** is there any configuration for localhost too?

**suggestion:** if there is something to do for local testing, I think it would deserve a callout for our default localhost:4000 example URL

(same for Google)